drivers/regulator/regulator_axp2101.c

```diff
@@ -0,0 +1,460 @@
+/*
```

Comment:
Please unify your implementation with axp192.
(Make sure to support axp2101 in the regulator_axp192.c file)
This device is very feature-rich, and duplicating drivers at every layer will make maintenance a pain.